<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kodeiklanLabel">
      <property name="text">
       <string>kode iklan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodeiklanLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="produkLabel">
      <property name="text">
       <string>Produk</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="produkLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="brandLabel">
      <property name="text">
       <string>Brand</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="brandLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="versiLabel">
      <property name="text">
       <string>Versi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="versiLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="durasiLabel">
      <property name="text">
       <string> Durasi (menit)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="durasiLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="jenisLabel">
      <property name="text">
       <string>Jenis</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="jenisLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="hargaLabel">
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="hargaLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="hapusButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="simpanButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>410</y>
     <width>551</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Kode Iklan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Produk</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Brand</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Versi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Durasi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jenis</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Iklan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ubahButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
